pkg/install/install.go

```diff
@@ -22,6 +22,7 @@ func CmdInstall() *cobra.Command {
 		Short: "Install the operator and create the noobaa system",
 		Run:   RunInstall,
 	}
+	cmd.Flags().Bool("skip_graceful_finalizer", false, "Create NooBaa system witout graceful finalizer")
```

Comment:
style: how about renaming it to `no_graceful_finalizer` mainly because it's hard to imagine what does skipping over a graceful finalizer actually means - sounds like poetry though...

Reply:
accepted :)